task3_snkrs/spiders/lanebryant_spider.py

```diff
@@ -0,0 +1,116 @@
+import json
+
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+from w3lib.url import add_or_replace_parameters
+
+from task3_snkrs.items import LanebryantItem
+
+
+class LanebryantSpider(CrawlSpider):
+    name = "lanebryant"
+
+    allowed_domains = [
+        'lanebryant.com',
+        'lanebryant.scene7.com',
+    ]
+    start_urls = [
+        'https://www.lanebryant.com',
+    ]
+    listings_css = [
+        '.mar-subnav-links-column',
+        '.mar-pagination a',
+    ]
+    rules = [
+        Rule(LinkExtractor(restrict_css=listings_css)),
+        Rule(LinkExtractor(restrict_css='.mar-prd-item-image-container'), callback='parse_product'),
+    ]
+
+    def parse_product(self, response):
+        data = json.loads(response.css('#pdpInitialData::text').getall()[0])
+        data = data['pdpDetail']['product'][0]
+
+        item = LanebryantItem()
+        item['product_id'] = self.get_product_id(data)
+        item['name'] = self.get_name(data)
+        item['brand'] = self.get_brand()
+        item['description'] = self.get_description(response)
+        item['skus'] = self.get_skus(data)
+        item['meta'] = {'requests': self.image_requests(response, data, item)}
+
+        return self.request_or_item(item)
+
+    def parse_image_urls(self, response):
+        data = response.meta['data']
+        item = response.meta['item']
+
+        if item.get('image_urls'):
```

Comment:
Let's set `image_urls` as empty list in `parse` method, so that this check will not be needed any more.